src/App.js

```diff
@@ -1,13 +1,22 @@
 import React from "react";
-
+import Heading from "./Heading";
+import TouristInfoCards from "./TouristInfoCards";
 import Bookings from "./Bookings";
+import Footer from "./Footer";
 import "./App.css";
 
 const App = () => {
+  const addresses = [
+    "123 Fake Street, London, E1 4UD",
+    "hello@fakehotel.com",
+    "0123 456789"
+  ];
   return (
     <div className="App">
-      <header className="App-header">CYF Hotel</header>
+      <Heading />
+      <TouristInfoCards />
       <Bookings />
+      <Footer keys={addresses} />
```

Comment:
Keys is not the most descriptive name for what this prop does - can you think of a name which better describes what the values are / how they'll be used?